src/tidysic.py

```diff
@@ -41,7 +45,23 @@ def parse_in_directory(dir_src, with_album):
         artist = tag.artist
         title = tag.title
 
-        if artist:
+        if guess and not artist or not title:
```

Comment:
Even if this works as intended, parentheses don't hurt for lisibility

Reply:
```suggestion
        if guess and (not artist or not title):
```

Reply:
I agree with you Herr Soßekönig